### DISCUSSION TITLE
RuntimeError: (NotFound) The kernel `sync_batch_norm` is not registered.   [Hint: Expected iter != kernels_.end(), but received iter == kernels_.end().] (at /paddle/paddle/phi/core/kernel_factory.cc:174)

### DISCUSSION THREAD

**Discussion by vani-mcm** [PaddlePaddle/PaddleOCR#12864] [Q&A] [answered]
I am currently working with fine tuning of SLANET with custom table dataset. Prepared annotations using PaddleLabel tool.
While training SLANET with the text file generated from PaddleLabel tool, I get the above error
I tried conda environment in python 3.7, 3.8, 3.9 and 3.10
OS: ubuntu
paddlepaddle : 2.5.1
![errgh](https://github.com/PaddlePaddle/PaddleOCR/assets/132423482/e4bb78c7-2135-40d1-b02c-7096cc3b3a79)
Pls help me.
If any further information needed, I will provide.
I really need help.

**Comment by SWHL** [comment]
If the problem still exists, please try to to update Paddle and PaddleOCR version to latest

**Comment by wang-love-yu** [comment, marked as answer]
参考 (https://github.com/PaddlePaddle/PaddleOCR/issues/7589)
配置文件 use_sync_bn: True，修改为False，sync_batch_norm 不支持cpu
